run/deployment-previews/README.md

```diff
@@ -1,25 +1,522 @@
+[Cloud Shell](https://console.cloud.google.com/?cloudshell=true), or your local
+machine if you have installed and configured `git` and Google Cloud CLI. The
+instructions below show both methods.
+
+TODO(glasnt0: cleanup)
+
+* Configure Google Cloud CLI to use your project, replacing <code><var>MYPROJECT</var></code>
+  with your project ID:
+
+    ```
+    export PROJECT_ID=MYPROJECT
+    gcloud config set project $PROJECT_ID
+    ```
+
+    If using Cloud Shell, you may need to authorize Google Cloud CLI to make a
+    Google Cloud API call. Click <b>Authorize</b> to allow this action to proceed.
+* Create a new container image:
 
 ```
-docker run --rm deployment-previews
+git clone https://github.com/GoogleCloudPlatform/python-docs-samples
+cd python-docs-samples/
+gcloud builds submit --tag gcr.io/$PROJECT_ID/deployment-previews run/deployment-previews
 ```
 
-## Test
+* Confirm that the container has been created:
 
 ```
-pytest
+gcloud container images list
+```
+* Remove the cloned repository:
+```
+cd ..
+rm -rf python-docs-samples
 ```
+### Add the new Cloud Build configuration
+
+Your repository already has a `cloudbuild.yaml` file that is used on your main
+branch. You will now create a new configuration for this new trigger.
+
+1. On your GitHub repo page, click **Add file** > **Create new file**
+    1. Name the new file `cloudbuild-preview.yaml`
+    1. Copy the code into your new file: [cloudbuild-preview.yaml](/run/deployment-previews/cloudbuild-configurations/cloudbuild-preview.yaml)
+2. Commit the change to the main branch of your repository.
+
+
+## Creating the secondary trigger
+
+Now that all the groundwork is in place, create the new trigger.
+
+1. Create a new Cloud Build trigger ([full instructions](https://cloud.google.com/build/docs/automating-builds/create-manage-triggers#console)):
+    1. In the Google Cloud console, go to the **Cloud Build triggers** page.
+
+       <a href="https://console.cloud.google.com/cloud-build/triggers"
+           target="console"
+           track-type="tutorial"
+           track-name="consoleLink"
+           track-metadata-position="body"
+           class="button button-primary">Go to the Cloud Build triggers page</a>
+
+    1. Click **Create Trigger**.
+    1. Enter the following details:
+      *   Name: `preview-deploy`
+      *   Event: Pull Request
+      *   Source Repository: "<var>GITHUB_USER_NAME</var>/helloworld-python"
+      *   Base branch: `^main$`
+      *   Comment control: Required except for owners and collaborators
+         -  As the owner of the repo, previews will be automatically built on pull requests you create.
+         -  If you want to allow anyone to preview their changes, read more about the [security implications](https://cloud.google.com/build/docs/cloud-build-service-account) of selecting "Not required".
+      *   Configuration: Cloud Build Configuration file
+      *   Cloud Build configuration file location: `cloudbuild-preview.yaml`
```

Comment:
Check if you are happy with how this is rendering in Markdown

Reply:
Corrected ✨